src/url-params.ts

```diff
@@ -189,16 +195,29 @@ export class AnalyticsUrlParams {
 			actualDomainMatch = null;
 		}
 
-		const destinationDomainMatch = destinationUrl
-			? destinationUrl.hostname.match(regex)
-			: null;
+		actualDomain = actualDomainMatch ? actualDomainMatch.toString() : null;
+		if (typeof destinationUrl === 'string') {
+			try {
+				destinationUrl = new URL(destinationUrl);
+				destinationDomainMatch = destinationUrl
+					? destinationUrl.hostname.match(regex)
+					: null;
+
+				destinationDomain = destinationDomainMatch
+					? destinationDomainMatch.toString()
+					: null;
+			} catch (err) {
+				return ''; //I dont know what we should do when an error like this happens. Any ideas??
```

Comment:
Good question. We could do `console.log(err)` but I don't know how much that'd help. I think it's fine to at least catch it as you do here.

Reply:
@thgreasi @iamsolankiamit any suggestions on handling such caught errors in modules? 

It's OK to silently fail here but I wonder what's the best practice.